app/javascript/pages/MyEvents/CreateEditModalContents.jsx

```diff
@@ -0,0 +1,174 @@
+
+import { Button } from '@zendeskgarden/react-buttons'
+import styled from 'styled-components'
+
+const PaddedField = styled(GField)`
+  flex: 1;
+  padding: ${({ theme }) => theme.space.sm} ${({ theme }) => theme.space.xs};
+`
+
+const TopMargin = styled.div`
+  margin-top: ${({ theme }) => theme.space.xs};
+`
+const RightFloatButton = styled(Button)`
+  float: right;
+`
+
+const CreateEditModalContents = ({
+  offices,
+  eventTypes,
+  tags,
+  organizations,
+  createEditIndividualEvent,
+  isNew,
+  setShowCreateEditModal,
+  modalEventData,
+}) => {
+  const currentUserOffice = useContext(UserContext).currentUser.office
+
+  const [description, setDescription] = useState(modalEventData?.description || '')
+  const [selectedTags, setSelectedTags] = useState(modalEventData?.tags || [])
+  const [selectedOffice, setSelectedOffice] = useState(modalEventData?.office || currentUserOffice.id)
+  const [date, setDate] = useState(modalEventData?.date || new Date())
+  const [duration, setDuration] = useState(modalEventData?.duration || 0)
+  const [selectedEventType, setSelectedEventType] = useState(modalEventData?.eventType)
+  const [selectedOrg, setSelectedOrg] = useState(modalEventData?.organization)
+
+  const [showFieldErrors, setShowFieldErrors] = useState(false)
+  const [loading, setLoading] = useState(false)
+
+  const isValidText = value => (value.length === 0 ? false : true)
+  const isValidMultiSelect = selection => (selection.length < 1 ? false : true)
+  const isValidSelect = selection => !!selection
+  const isValidNumber = number => (number > 0 ? true : false)
+
+  const handleSubmit = () => {
+    if (
+      isValidText(description) &&
+      isValidMultiSelect(selectedTags) &&
+      isValidSelect(date) &&
+      isValidNumber(duration) &&
+      isValidSelect(selectedEventType) &&
+      isValidSelect(selectedOrg)
+    ) {
+      const data = {
+        id: modalEventData?.id,
+        description,
+        tags: selectedTags,
+        officeId: selectedOffice,
+        date: date.toLocaleString(),
+        duration,
+        eventTypeId: selectedEventType,
+        organizationId: selectedOrg,
+      }
+      setLoading(true)
+      createEditIndividualEvent(data).then(() => {
+        setLoading(false)
+        setShowCreateEditModal(false)
+      })
+    } else {
+      setShowFieldErrors(true)
+    }
+  }
+
+  return (
+    <>
+      <Header>{isNew ? 'Record Event' : 'Edit Event'}</Header>
+      <Body>
+        <PaddedField>
+          <Label>Description</Label>
+          <Input value={description} onChange={event => setDescription(event.target.value)} />
+          {showFieldErrors && !isValidText(description) && (
+            <Message validation={'error'}>{'Must not be empty'}</Message>
+          )}
+        </PaddedField>
+        <PaddedField>
+          <Label>Organization</Label>
+          <Hint>Contact us to add your organization to this list or select 'Other'</Hint>
```

Comment:
How do they contact us?

Reply:
![image](https://user-images.githubusercontent.com/27116427/81134113-0ed69900-8f97-11ea-96fb-a281b37e5d3a.png)

We've got the support widget. Not everyone will have it but I suspect there'll be a point of contact (the admins) in which they can add organisation. This will be how it is for now until volunteers can add organizations.